get latest portal bundle url

lawrence-lee opened the pull request.

lawrence-lee committed:
get latest portal bundle

lawrence-lee committed:
simplify task

lrbladejenkins commented:
Bundles Diff Test failed on LAX linux server on 75319626f5c84a5fa76e0bcb162d942e0d495a75

[link to jenkins](http://cloud-10-50-0-245:8081/job/liferay-blade-samples-pr-tester/789/)

[Console output](http://cloud-10-50-0-245:8081//job/liferay-blade-samples-pr-tester/789/consoleText)

lawrence-lee commented:
```FAILURE: Build failed with an exception.

* Where:
Build file '/home/jenkins/.jenkins/workspace/liferay-blade-samples-pr-tester/build.gradle' line: 187

* What went wrong:
Execution failed for task ':bundlesDiffTest'.
> [directory 'maven' - com.liferay.blade.basic.service-1.0.0.jar 
  CHANGED    MANIFEST   <manifest>
   REMOVED    HEADER     Provide-Capability:osgi.service;objectClass:List<String>="com.liferay.blade.samples.servicebuilder.service.FooLocalService",osgi.service;objectClass:List<String>="com.liferay.blade.samples.servicebuilder.service.FooService"
  CHANGED    RESOURCES  <resources>
   REMOVED    RESOURCE   META-INF/service.xml
  , directory 'maven' - com.liferay.blade.samples.jndiservicebuilder.service-1.0.0.jar 
  CHANGED    MANIFEST   <manifest>
   REMOVED    HEADER     Provide-Capability:osgi.service;objectClass:List<String>="com.liferay.blade.samples.jndiservicebuilder.service.RegionLocalService"
  CHANGED    RESOURCES  <resources>
   REMOVED    RESOURCE   META-INF/service.xml
  , directory 'maven' - com.liferay.blade.samples.jdbcservicebuilder.service-1.0.0.jar 
  CHANGED    MANIFEST   <manifest>
   REMOVED    HEADER     Provide-Capability:osgi.service;objectClass:List<String>="com.liferay.blade.samples.jdbcservicebuilder.service.CountryLocalService"
  CHANGED    RESOURCES  <resources>
   REMOVED    RESOURCE   META-INF/service.xml
  ]

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.```

lawrence-lee commented:
@gamerson , can you merge this still? I'll have to investigate this error some.

lrbladejenkinswindows commented:
Bundle Diff Test failed on LAX windows server on 75319626f5c84a5fa76e0bcb162d942e0d495a75

[link to jenkins](http://cloud-10-50-0-254:80/job/liferay-blade-samples-pr-tester/462/)

gamerson commented:
Pull request submitted to https://github.com/gamerson/liferay-blade-samples/pull/441. [See changes here.](https://github.com/gamerson/liferay-blade-samples/compare/0c24c48~1...0c24c48) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*
